htmlvalidator/middleware.py

```diff
@@ -1,5 +1,8 @@
 from django.conf import settings
-from django.contrib.sites.models import RequestSite
+try:
+    from django.contrib.sites.requests import RequestSite
+except Importerror:  # Django < 1.7
```

Comment:
Shouldn't that comment be `# Django < 1.8`. I.e. you're in the `ImportError` block if you're NOT running django 1.8